### ISSUE TITLE
Incorporating EMMET, a new model editing algorithm in EasyEdit

### ISSUE THREAD

**Issue by akshat57** [zjunlp/EasyEdit#263]
Hi,

Recently we released a new model editing algorithm - EMMET or Equality-Constraint Mass Model Editing Algorithm for Transformers, which generalizes ROME to the batch setting. This essentially allows making batched edits using the ROME loss function. 

The details of our work in present in our paper - https://arxiv.org/abs/2403.14236
And our github repo is this - https://github.com/scalable-model-editing/unified-model-editing

For the last month or so we've been working on stabilizing EMMET and finding the optimal hyperparameters. Now we can confidently say that it works atleast as well as MEMIT and in the paper we do present a few more ideas. 

We would be grateful if our work can be incorporated in your repository and framework. We are happy to submit a PR if that makes things easier. 

Thank,
Akshat

**Comment by XeeKee** [comment]
Hello Akshat,

Thank you very much for your interest in EasyEdit. We will schedule support for it in the near future.

**Comment by pengzju** [comment]
Thx, feel free to submit PRs at any time. 👏

**Comment by akshat57** [comment]
Hi, 

We just submitted a PR for incorporating EMMET into EasyEdit. - https://github.com/zjunlp/EasyEdit/pull/334

Excited to finally have EMMET become part of the EasyEdit family. 

Thanks,
Akshat

**Comment by zxlzr** [comment]
Hi Akshat,

Thank you very much, we will check and merge it ASAP.

Best,

EasyEdit Team

**Comment by XeeKee** [comment]
Hello Akshat, thank you very much for your interest in easyedit. We have successfully merged your pull request into our repository. We appreciate it!
